inspired by qhull-2003.1-pkgconfig.patch, available from git://pkgs.fedoraproject.org/qhull
requires making CMake to fill in the gaps via 'configure_file(... @ONLY)'
author: Kenneth Hoste (UGent)
diff -Naur qhull/qhull.pc.in qhull/qhull.pc.in
--- qhull/qhull.pc.in	1970-01-01 01:00:00.000000000 +0100
+++ qhull/qhull.pc.in	2012-07-08 08:04:07.795336862 +0200
@@ -0,0 +1,11 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+includedir=${prefix}/include
+libdir=${prefix}/lib
+
+Name: qhull
+Description: General dimension convex hull library
+Version: @qhull_VERSION2@
+Requires:
+Libs: -L${libdir} -lqhull -lm
+Cflags: -I${includedir}
